src/app/state/transaction/types.ts

```diff
@@ -1,26 +1,22 @@
 import { ErrorPayload } from 'types/errors'
 
-/**
- * These are manually copied from Oasis-explorer. Later, oasis-explorer should
- * make those an enum so that we maintain strong typing across projects.
```

Comment:
does anything rely on these matching up with oasis-explorer?

Reply:
No; we map from oasis-explorer's OperationsRowTypeEnum to our TransactionType here: https://github.com/oasisprotocol/oasis-wallet-web/blob/825bac2d0ad22132032c69cf93615de65ca9fc16/src/vendors/monitor.ts#L123-L140

Oasis-explorer's OperationsRowTypeEnum is generated from https://github.com/oasisprotocol/oasis-wallet-web/blob/825bac2d0ad22132032c69cf93615de65ca9fc16/src/vendors/explorer/swagger.yml#L1125-L1141

and we have this test to try to ensure the enum in swagger.yml is accurate https://github.com/oasisprotocol/oasis-wallet-web/blob/825bac2d0ad22132032c69cf93615de65ca9fc16/cypress/integration/api-transaction-methods.ts#L15-L32